tests/typecheck/db/models/test_constraints.yml

```diff
@@ -35,3 +35,13 @@
         main:4: note: Possible overload variants:
         main:4: note:     .*
         main:4: note:     .*
+
+-   case: unique_constraint_nulls_distinct
```

Comment:
Test is not required, can you please remove it? They are really slow :(

Reply:
Works for me :+1: